scripts/deploy-kubefed.sh

```diff
@@ -178,9 +174,6 @@ cd -
 # Deploy KubeFed resources
 deploy-with-helm
 
-# Wait for admission webhook server to be ready
-util::wait-for-condition "kubefed admission webhook to be ready" "kubefed-admission-webhook-ready ${NS}" 120
```

Comment:
I am assuming, we are removing this because the helms `--wait` does this for us now.

Reply:
Correct.